fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPage.kt

```diff
@@ -65,15 +75,19 @@ fun OnboardingPage(
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.SpaceBetween,
         ) {
-            IconButton(
-                onClick = onDismiss,
-                modifier = Modifier.align(Alignment.End),
-            ) {
-                Icon(
-                    painter = painterResource(id = R.drawable.mozac_ic_close),
-                    contentDescription = stringResource(R.string.onboarding_home_content_description_close_button),
-                    tint = FirefoxTheme.colors.iconPrimary,
-                )
+            if (onDismiss != null) {
+                IconButton(
+                    onClick = onDismiss,
+                    modifier = Modifier.align(Alignment.End),
+                ) {
+                    Icon(
+                        painter = painterResource(id = R.drawable.mozac_ic_close),
+                        contentDescription = stringResource(R.string.onboarding_home_content_description_close_button),
+                        tint = FirefoxTheme.colors.iconPrimary,
+                    )
+                }
+            } else {
+                Spacer(Modifier)
             }
```

Comment:
❓What does the Spacer do here, is it required?

Reply:
It's a dummy composable. Since the parent is using `verticalArrangement = Arrangement.SpaceBetween` and the design is such that the buttons must be at the bottom of the available screen space. This spacer acts as a first element in the parent in the absence of the close button.

Reply:
Gotcha, thanks for explaining